modulefiles/module_gwci.hera.lua

```diff
@@ -2,14 +2,14 @@ help([[
 Load environment to run GFS workflow setup scripts on Hera
 ]])
 
-prepend_path("MODULEPATH", "/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/modulefiles/stack")
+prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.1/envs/gsi-addon/install/modulefiles/Core")
 
-load(pathJoin("hpc", "1.1.0"))
-load(pathJoin("hpc-intel", "18.0.5.274"))
-load(pathJoin("hpc-impi", "2018.0.4"))
+load(pathJoin("stack-intel", os.getenv("2021.5.0")))
+load(pathJoin("stack-intel-oneapi-mpi", os.getenv("2021.5.1")))
 
-load(pathJoin("netcdf","4.7.4"))
-load(pathJoin("nccmp","1.8.7.0"))
+load(pathJoin("netcdf-c", os.getenv("4.9.2")))
+load(pathJoin("netcdf-fortran", os.getenv("4.6.0")))
+load(pathJoin("nccmp","1.9.0.1"))
```

Comment:
I would think the CI modulefiles also need the `py-yaml`, `py-jinja2` etc. from `gfs-workflow`.  They are loaded on the orion gwci modulefile, so not sure why they were missing here.

Reply:
Yeah, that's a good point.  I would imagine that they are needed.  I will do a grep survey of the CI python modules and see if there's anything else that might be needed.